RCSwitch.cpp

```diff
@@ -466,10 +465,12 @@ void RCSwitch::transmit(int nHighPulses, int nLowPulses) {
   }
 #endif
 
-  digitalWrite(this->nTransmitterPin, HIGH);
-  delayMicroseconds( this->protocol.pulseLength * nHighPulses);
-  digitalWrite(this->nTransmitterPin, LOW);
-  delayMicroseconds( this->protocol.pulseLength * nLowPulses);
+  for(uint8_t i = 0; i < COUNT_OF(sequence.pulses); i++)
```

Comment:
hmm... so we now have a loop... with two iterations... but to see that, and understand the code, one has to read the headerfile. And know what this non-standard `COUNT_OF` macro does. This is really a bit overengineered :)